contrib/gogearbox/gearbox.v1/gearbox_test.go

```diff
@@ -0,0 +1,280 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package gearbox
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gogearbox/gearbox"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Test that the gearbox request context retains the tracer context
+// func TestChildSpan(t *testing.T) {
```

Comment:
Leaving this tests commented out, because I'm not sure how to approach them when the go server is siphoned away from the tracer.